system-status.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Run this script to gather details about the environment where the CI job is
+# running. This can be helpful to identify issues why minikube failed to
+# deploy, or tests encounter problems while running.
+#
+
+function minikube_ssh() {
+    ssh \
+        -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no \
+        -l docker -i "$(minikube ssh-key)" \
+        "$(minikube ip)" "${*}"
+}
+
+function log() {
+    echo "###"
+    echo "### going to execute: ${*}"
+    echo "###"
+    ${*}
```

Comment:
with quotes the executable gets interpreted like `virsh status` instead of `virsh` with `status` as argument. Will check what is accepted by ShellCheck and works.